app/controllers/schools/consents_controller.rb

```diff
@@ -8,7 +8,7 @@ def show
     end
 
     def create
-      @consent_grant = ConsentGrant.new(school_consent_params.merge(user: current_user, school: @school))
+      @consent_grant = ConsentGrant.new(school_consent_params.merge(user: current_user, school: @school, ip_address: request.remote_ip))
```

Comment:
I think this will be incorrect on the test/live servers because we're behind an Elastic Load Balancer, needs to use X-Forwarded-For I think to be the actual user IP.